storage-migration-error.asc

```diff
@@ -0,0 +1,54 @@
+:docinfo1:
+= Unexpected error during Storage Migration
+:sectanchors:
+
+There was an error copying your media to a folder supported by Android R. Before the process started, your data was backed up to AnkiWeb and can be restored.
+
+Before continuing, please:
+
+1. Take a backup of your collection (including scheduling + media) via Deck Picker -> Export collection
+  * See <<take_manual_backup,'Manual Backups'>> if you desire a file-based backup
+2. Report the error to us: https://docs.ankidroid.org/help.html#_support
+
+=== Restoring from AnkiWeb Backup
+
+If you have reviewed a significant number of cards using AnkiDroid since the migration started, see: <<restore_modified_collection,Restoring a Modified Collection>>
+
+* Reinstall AnkiDroid
+** Do not keep app data on uninstall. (*Ensure you have backed up, this will delete the AnkiDroid collection, media and backups from the AnkiDroid folder*)
+* Open AnkiDroid, select "Sync from AnkiWeb", 
+** Log in and re-sync your collection
+** Syncing will be significantly faster than AnkiDroid 2.15
+* Once you are synced, you may delete the folder collection in `/storage/emulated/0/AnkiDroid` to save space. 
+* Your collection is now located at `/storage/emulated/0/Android/data/com.ichi2.anki/files`
+
+=== Restoring a Modified Collection [[restore_modified_collection]]
+
+*Ensure that you have backed up your collection and media*
+
+* Uninstall AnkiDroid
+** Do not keep app data. (**This will delete the AnkiDroid collection, media and backups**)
```

Comment:
This is quite confusing, first you advise to backup things and then to delete everything, including the backups, plus again, how do you export a collection that is split between folders?